packages/astro/src/@types/astro.ts

```diff
@@ -95,12 +175,29 @@ export interface AstroGlobalPartial {
 	/**
 	 * @deprecated since version 0.24. See the {@link https://astro.build/deprecated/resolve upgrade guide} for more details.
 	 */
-	resolve: (path: string) => string;
-	/** @deprecated Use `Astro.glob()` instead. */
+	resolve(path: string): string;
+	/** @deprecated since version 0.26. Use [Astro.glob()](https://docs.astro.build/en/reference/api-reference/#astroglob) instead. */
 	fetchContent(globStr: string): Promise<any[]>;
+	/**
+	 * Fetch local files into your static site setup
+	 *
+	 * Example usage:
+	 * ```typescript
+	 * const posts = await Astro.glob('../pages/post/*.md');
+	 * ```
+	 *
+	 * [Astro documentation](https://docs.astro.build/en/reference/api-reference/#astroglob)
+	 */
 	glob(globStr: `${any}.astro`): Promise<ComponentInstance[]>;
 	glob<T extends Record<string, any>>(globStr: `${any}.md`): Promise<MarkdownInstance<T>[]>;
 	glob<T extends Record<string, any>>(globStr: string): Promise<T[]>;
+	/**
+	 * Returns an [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) object built from the [site](https://docs.astro.build/en/reference/configuration-reference/#site) config option
```

Comment:
I debated commenting on this this morning, since this takes us back into the "an .astro file" discussion 😅... but just checking that "an URL" is a conscious choice. 

(I tend to say U-R-L, which means I'd naturally write "a URL" but can't guarantee that's how everyone rolls!)

Reply:
Haha! I thought about that debate while writing it. I do always (consciously) write `an URL` because I tend to pronounce URL in French, even in English sentences

I do believe that the proper way English way is `a URL` though, I'm not picky on this, I'll go with the consensus

Reply:
Sometimes, all we can do is try to predict the option that will make the fewest people react, thinking it's "wrong"...  and that's not always a winnable game.  :)

I would go with `a URL` if I were aiming at an English-speaking audience, myself. But for an international audience, my preference is slighter, and neither option bothers me. Just wouldn't be doing my job if I didn't bring it up!